lib/replicache.dart

```diff
@@ -356,9 +363,17 @@ class Replicache implements ReadTransaction {
     // One initial call.
     yield await query(callback);
 
-    await for (final _ in _changeStreamController.stream) {
-      yield await query(callback);
-    }
+    _Subscription subscription;
+    // ignore: close_sinks
+    StreamController<R> streamController = StreamController(
+      onListen: () => _subscriptions.add(subscription),
+      onCancel: () => _subscriptions.remove(subscription),
```

Comment:
This seems wrong related to close. Will fix.

Reply:
I'm really not sure how the stream api is supposed to work. I think this was right after all.

Reply:
I'm happy enough to move on...